.github/workflows/release.yml

```
@@ -12,7 +12,7 @@ jobs:
       contents: read
     runs-on: ubuntu-latest
     env:
-      NX_CLOUD_ACCESS_TOKEN: ${{ secrets.NX_CLOUD_ACCESS_TOKEN }}
+      NX_CLOUD_ACCESS_TOKEN: ${{ env.NX_CLOUD_ACCESS_TOKEN }}
```

```suggestion
      NX_CLOUD_ACCESS_TOKEN: ${{ github.event_name == 'push' && github.ref != 'main' && '' || secrets.NX_CLOUD_ACCESS_TOKEN }}
```
Because this workflow is not a `workflow_call`

---

maybe for this one we don't need to provide the `NX_CLOUD_ACCESS_TOKEN`
and it will be enough to disable nx cloud for releases as we are in the `release` workflow